flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -48,10 +48,15 @@ function setup_kubernetes_for_linux {
       sudo rm "$(which minikube)"
     fi
 
+    # Retry loop download minikube.
```

Comment:
This comment doesn't add any additional value. The code below makes it quite clear that retries are done. I guess we can remove the comment again.